ansible/roles/cuda/defaults/main.yml

```diff
@@ -1,7 +1,6 @@
-cuda_distro: "rhel{{ ansible_distribution_major_version }}"
```

Comment:
why was this removed despite its templated value being duplicated elsewhere? Maybe private var instead if we don't want it to be overridden